index.js

```diff
@@ -129,91 +102,80 @@ const HOME_MENU_BUTTONS_ARRAY = [
     {
         id: ELEMENT_IDS.digitalProgrammeListTab,
         innerText: "Digital Programme List",
-        onclick: renderDigitalProgrammeList
+        onclick: () => console.log(innerText)
     },
     {
         id: ELEMENT_IDS.digitalEpgTab,
         innerText: "Digital EPG",
-        onclick: renderDigitalEpg
+        onclick: () => console.log(innerText)
     },
     {
         id: ELEMENT_IDS.applicationsTab,
-        param2: "applications",
+        buttonName: "applications",
```

Comment:
Instead of introducing new key `innerText` could be transformed

Reply:
Don't really know how you mean. I need buttonName to handle disable buttons.